### ISSUE TITLE
 Lychee 3.1

### ISSUE THREAD

**Issue by innerhippy** [GustavePate/lycheesync#76]
I get the impression that this project is no loner maintained, which is a shame. Doesn't seem to work with the latest version of Lychee (3.1). Can anyone confirm this or provide alternatives?

**Comment by gslavov** [comment]
While it looks abandoned, it works fine with the latest version of Lychee (3.1.6) on my system (CentOS 7).

In saying that, there are a few things that don't work out of the box. I used a prefix for my lychee tables so had to modify the following files:
-lycheesync/lycheedao.py
-lycheesync/update_scripts/inf_to_lychee_2_6_2.py
-ressources/lychee.sql
-tests/test_main.py
-tests/testutils.py

just doing a basic search and replace (ie: sed -i 's/lychee_albums/lyc_lychee_albums/g' lychee.sql) for the four tables in the files listed above.

I am not sure if it was required or not but I also replaced the INSERT statement for the settings table in lychee.sql with the correct values (as in, changed the version, username, pass and identifier to the values in the actual database)
